packages/web3-common/src/eth_execution_api.ts

```diff
@@ -158,6 +158,7 @@ export interface ReceiptInfo {
 	readonly logsBloom: HexString256Bytes;
 	readonly root: HexString32Bytes;
 	readonly status: '0x1' | '0x0';
+	readonly effectiveGasPrice: Uint;
```

Comment:
Should not it be an optional attribute?

Reply:
In [execution apis specs](https://github.com/ethereum/execution-apis/blob/main/src/schemas/receipt.json#L44) is referred as required. Do I miss something?